src/HW/server/DB/migrations/20220621125709-create-item.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
```

Comment:
```suggestion
    await queryInterface.createTable('items', {
```
table name convention